admin/src/main/resources/io/buoyant/admin/js/src/combined_client_graph.js

```diff
@@ -89,6 +103,14 @@ define([
           ignoredClients[routerName][client] = false;
         },
 
+        expireClient: function(client) {
+          expiredClients[routerName][client] = true;
```

Comment:
that is definitely possible - however, if we keep track of them we can put them back in their original color (both the client section and the combined graph) - which is possibly less confusing than picking a new color every time this client is unexpired. I guess we don't really know how likely it is that clients will come back, or go for good